code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -521,7 +521,13 @@
 	. = ..()
 	if(.)
 		return
-	return (sunder * -0.01) + 1
+	return (sunder * -0.01) + 1 
+
+/mob/living/carbon/xenomorph/get_sunder_multiplier()
```

Comment:
What's the use case where you'd want more than just the last line here?

Reply:
What do you mean?